FindOnPage.md

```diff
@@ -0,0 +1,478 @@
+    
+        return true;
+    }
+    //! [GetActiveMatchIndex]
+```
+#### .NET C#
+```csharp
+//! [GetActiveMatchIndex]
+public async Task<int> GetActiveMatchIndexAsync()
+{
+    var webViewFind = webView.CoreWebView2.FindController; // Assuming webView is your WebView2 control
+    var activeMatchIndex = await webViewFind.GetActiveMatchIndexAsync();
+    MessageBox.Show($"Active Match Index: {activeMatchIndex}", "Find Operation", MessageBoxButton.OK);
+    return activeMatchIndex;
+}
+
+//! [GetActiveMatchIndex]
+```
+
+
+
+## API Details
+```csharp
+
+/// Specifies the direction of find Parameters.
+[v1_enum]
+typedef enum COREWEBVIEW2_FIND_DIRECTION { 
+    /// Specifies a forward find in the document.
+    COREWEBVIEW2_FIND_DIRECTION_FORWARD, 
+    /// Specifies a backwards find in the document.
+    COREWEBVIEW2_FIND_DIRECTION_BACKWARD 
+} COREWEBVIEW2_FIND_DIRECTION; 
+
+// Interface defining the find configuration.
+[uuid(4A6ED732-DF08-4449-8949-3632A4DEBFCD), object, pointer_default(unique)] 
+interface ICoreWebView2FindConfiguration : IUnknown {
+
+    // Gets or sets the find term used for the find operation.
+    [propget] HRESULT FindTerm([out, retval] LPWSTR* value);
+    [propput] HRESULT FindTerm([in] LPCWSTR value);
+
+    // Gets or sets the direction of the find operation (forward or backward).
+    [propget] HRESULT FindDirection([out, retval] COREWEBVIEW2_FIND_DIRECTION* value); 
+    [propput] HRESULT FindDirection([in] COREWEBVIEW2_FIND_DIRECTION value);
+
+    // Gets or sets whether the find operation is case sensitive.
+    [propget] HRESULT IsCaseSensitive([out, retval] BOOL* value); 
+    [propput] HRESULT IsCaseSensitive([in] BOOL value);
+
+    // Gets or sets whether to only match whole words during the find operation.
+    [propget] HRESULT ShouldMatchWord([out, retval] BOOL* value); 
+    [propput] HRESULT ShouldMatchWord([in] BOOL value);
+
+    // Gets the state of whether all matches are highlighted.
+    // Returns TRUE if all matches are highlighted, FALSE otherwise.
+    [propget] HRESULT ShouldHighlightAllMatches([out, retval] BOOL* value); 
+    // Sets the state to either highlight all matches or not.
+    [propput] HRESULT ShouldHighlightAllMatches([in] BOOL value);
+ 
+    // Checks if a custom user interface is desired by end developer.
+    // If TRUE, the default Find bar is not displayed.
+    // Returns TRUE if using a custom UI, FALSE if using the default.
+    [propget] HRESULT SuppressDefaultDialog([out, retval] BOOL* value); 
+    // Sets whether to use a custom UI for the Find operation.
+    [propput] HRESULT SuppressDefaultDialog([in] BOOL value);
+
+    // Gets the index of the currently active match.
+    // If there's no active find session but an attempt is made to get the active match index:
+    // The function will return -1.
+    // Returns The active match index.
+    [propget] HRESULT ActiveMatchIndex([out, retval] LONG* value); 
```

Comment:
In the MIDL3 these are named GetActiveMatchIndexAsync but are sync methods, here and in the C++ sample it is a property not a method, and in the C# sample code it is an actual async method. If this doesn't have to be async then it would be good to make it a property. If it must be async then the Get...Async style method looks good but should be consistent across COM IDL, MIDL3, and C++/C# samples.